13-collabs.Rmd

```diff
@@ -5,6 +5,248 @@
+
+Tell a short story to explain what lead to you making these changes including attempting to answer these questions:  
+
+- What is the problem that these changes will solve?
+- Do you have any URLs relevant issues or files you can share?
+- What inspired you to take this approach -- are there other things you tried?
+- Are there other pull requests related to this change?
+
+### Includes an explicit request for what kind of feedback is needed
+
+What would you like your reviewer to do with this pull request? Stating this explicitly can save both of you time in this code review.
+
+- Are you still in the early stages and looking for a bigger picture review? Let them know that before they waste their time digging into the code line-by-line.
+- Are you in the later stages and looking for detailed nit-picky review?
+- Are you looking for feedback on the results or methods?
+
+### Points out questionable areas that need extra attention
+
+Are there specific areas of the code you are having trouble with or are unsure about? Send a link to the [specific lines in GitHub](https://stackoverflow.com/questions/23821235/how-to-link-to-specific-line-number-on-github) you are asking about. Are there results that are surprising, confusing, or [smell wrong](https://github.com/jennybc/code-smells-and-feels#code-smells-and-feels)?  
+
+Be sure to detail what you have dug into and tried at this point for any problematic points.   
+
+### Are relatively small and focused
+
+Try to make sure your pull requests aren't too long! Code reviewing fatigue is very real. If you send a reviewer thousands of lines of code to review it will be very overwhelming to review or understand.
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">10 lines of code = 10 issues.<br><br>500 lines of code = &quot;looks fine.&quot;<br><br>Code reviews.</p>&mdash; I Am Devloper (@iamdevloper) <a href="https://twitter.com/iamdevloper/status/397664295875805184?ref_src=twsrc%5Etfw">November 5, 2013</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Alternatively, when you create a new branch try to set a very intentional (and relatively small) goal you would like to achieve with your upcoming pull request. Keeping your pull requests small and focused on one task at a time will not only help your reviewers but also will help yourself feel more accomplished and organized.
+
+Also recall that incremental changes are good! Perhaps you do have a very large restructuring of your repository you are trying to accomplish, but finding smaller reasonable sets of changes (which would each have their own pull requests) to reach that goal incrementally can help keep things more manageable.
+
+[Using Stacked Pull Requests in GitHub](https://blog.logrocket.com/using-stacked-pull-requests-in-github/)
```

Comment:
```suggestion
If you have changes that might be interdependent, you can make a pull request based on another existing pull request [using Stacked Pull Requests in GitHub](https://blog.logrocket.com/using-stacked-pull-requests-in-github/)
```

Reply:
I kinda wonder if we want an advanced tip style box for this kind of thing?

Reply:
If you like that idea I could make one

Reply:
I think "advanced" is relative. But maybe another thing to call it? Or maybe advanced is fine. "next steps" or something idk.

Reply:
This box is great! Thanks for adding!

Reply:
I can make an issue about maybe updating some of the boxes to a different style, but I think the notice option works for now :)

Reply:
see issue #53

Reply:
Awesome and agreed!